raft/raft.go

```diff
@@ -448,6 +452,12 @@ func (r *raft) tickHeartbeat() {
 		r.heartbeatElapsed = 0
 		r.Step(pb.Message{From: r.id, Type: pb.MsgBeat})
 	}
+
+	// If current leader cannot transfer leader in election timeout, become leader again.
```

Comment:
Shouldn't this be inside the `r.electionElapsed >= r.electionTimeout` block above at line 440?